figures/backfill.py

```diff
@@ -67,7 +67,7 @@ def backfill_enrollment_data_for_site(site):
             enrollment_data.append((obj, created))
         except CourseNotFound:
             errors.append('CourseNotFound for course "{}". '
```

Comment:
```suggestion
```

Reply:
@OmarIthawi Thanks, I've got a code cleanup effort to do. When I hit the pipeline logging, adding format vars and making sure all exceptions are tested will be part of that

Reply:
@OmarIthawi I saw I had a missing '{}' in the string, so adding vars to the format . Thanks!